deployments.json

```diff
@@ -1208,5 +1208,10 @@
     "name": "Bob",
     "chainId": 60808,
     "url": "https://explorer.gobob.xyz/address/0xcA11bde05977b3631167028862bE2a173976CA11"
+  },
+  {
+    "name": "Rupaya",
+    "chainId": 499,
+    "url": "https://rupayascan.io/address/0xca11bde05977b3631167028862be2a173976ca11"
```

Comment:
Thanks for the PR! This URL does not work for me, the even just rupayascan.io can't be reached. Can you resolve this and the merge conflict?

Reply:
Thank you for bringing this to my attention. I apologize for the oversight regarding the incorrect URL in the original pull request. I have now fixed the explorer URL in the latest commit. The updated URL should correctly point to the Rupaya blockchain explorer. Please review the changes and let me know if any further modifications are needed.

The corrected URL is now:
https://scan.rupaya.io/address/0xca11bde05977b3631167028862be2a173976ca11

I appreciate your patience and thank you for your thorough review. If you have any questions or need any additional information, please don't hesitate to ask.

Reply:
@devspan Can you fix the conflicts as well? Because this PR is from your `main` branch, I am unable to push a fix for the conflicts for you

Reply:
I hope the third time's the charm! I've made the necessary changes